src/components/Product.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+
+function Product(props) {
+	console.log(props)
+	return (
+		<div className="todo-item">
```

Comment:
In a real projects you should keep same indents through all files. Most projects have "eslint" tool which takes care about it automatically.

Reply:
Will try next time(file was deleted)